app/controllers/admin_routes/allPendingForms.py

```diff
@@ -816,3 +816,43 @@ def getNotesCounter():
     except Exception as e:
         print("Error selecting admin notes:", e)
         return jsonify({"Success": False}),500
+
+@admin.route('/admin/addToBanner/<form_id>', methods=['POST'])
+
+
+def submitToBanner(form_id):
```

Comment:
We already have a function that submits to banner, right? It happens whenever a form is approved by admins. We shouldn't be re-inventing that solution.